site/app/views/NavigationView.php

```diff
@@ -289,6 +289,7 @@ public function showGradeables($sections_to_list) {
                  </button>
 HTML;
 						}
+
```

Comment:
Get rid of this newline if it's accidental. If it's not, I'd argue against having the newline, because it makes it harder to tell which if-else logic block the `else` belongs to.